README.md

```diff
@@ -26,9 +26,9 @@ $ npm install basic-auth
 var auth = require('basic-auth')
 ```
 
-### auth(req)
+### auth(request)
```

Comment:
I would rather keep this to match the same naming in the source code so it matches in debuggers and such.